tests/ShapeFileTest.php

```diff
@@ -383,24 +384,24 @@ public static function shapesProvider(): array
         ];
 
         return [
-            [ShapeType::POINT, $pointsForPointType],
-            [ShapeType::POLY_LINE, $pointsForPolyLineType],
-            [ShapeType::POLYGON, $pointsForPolygonType],
-            [ShapeType::MULTI_POINT, $pointsForMultiPointType],
-            [ShapeType::POINT_Z, $pointsForPointType],
-            [ShapeType::POLY_LINE_Z, $pointsForPolyLineType],
-            [ShapeType::POLYGON_Z, $pointsForPolygonType],
-            [ShapeType::MULTI_POINT_Z, $pointsForMultiPointType],
-            [ShapeType::POINT_M, $pointsForPointType],
-            [ShapeType::POLY_LINE_M, $pointsForPolyLineType],
-            [ShapeType::POLYGON_M, $pointsForPolygonType],
-            [ShapeType::MULTI_POINT_M, $pointsForMultiPointType],
+            [ShapeType::Point->value, $pointsForPointType],
```

Comment:
You can pass a enum directly instead of an int.
```suggestion
            [ShapeType::Point, $pointsForPointType],
```